Model/NodeTranslation.php

```diff
@@ -0,0 +1,127 @@
+<?php
+declare(strict_types=1);
+
+namespace Snowdog\Menu\Model;
+
+use Magento\Framework\Model\AbstractModel;
+use Snowdog\Menu\Api\Data\NodeTranslationInterface;
+use Snowdog\Menu\Model\ResourceModel\NodeTranslation as NodeTranslationResource;
+
+class NodeTranslation extends AbstractModel implements NodeTranslationInterface
+{
+    /**
+     * @inheritDoc
+     */
+    protected function _construct()
+    {
+        $this->_init(NodeTranslationResource::class);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getTranslationId(): ?int
```

Comment:
I think a declaration of the function getTranslationId() is missing in the NodeTranslationInterface